pkg/doc/did/doc.go

```diff
@@ -433,7 +433,7 @@ func decodePublicKeyJwk(jwkMap map[string]interface{}, publicKey *PublicKey) err
 	}
 
 	publicKey.Value = pkBytes
-	publicKey.jsonWebKey = &jwk
+	publicKey.JSONWebKey = &jwk
```

Comment:
This will be set only when the rawDIDDoc has "publicKeyJwk". But when we create DIDDoc struct we set value and it internally converts to "publicKeyBase58" and we would be able to get back the "publicKeyJwk".

Reply:
btw - why aren't we using publicKeyJWK format internally instead of base58?